NitroxClient/Unity/Helper/AssetBundleLoader.cs

```diff
@@ -42,10 +42,22 @@ public static IEnumerator LoadUIAsset(string name, string canvasObjectName, bool
             assetRoot.SetParent(uGUI.main.screenCanvas.transform, false);
             UnityEngine.Object.Destroy(assetCanvas);
 
-            if (callback != null)
+            callback?.Invoke(assetRoot.gameObject);
+        }
+
+        public static bool HasAsset(string name)
+        {
+            return File.Exists(Path.Combine(assetRootFolder, name));
+        }
+
+        public static void DisableAsset(string name)
+        {
+            string oldPath = Path.Combine(assetRootFolder, name);
+            if (File.Exists(oldPath))
             {
-                callback.Invoke(assetRoot.gameObject);
+                File.Move(oldPath, oldPath + ".disabled");
             }
+
```

Comment:
You can use Unity's playerprefs to store information on the client. Moving files around should be avoided unless absolutely necessary because IO can easily fail due to PC configuration.